scripts/run_integration_tests_aarch64.sh

```diff
@@ -59,13 +59,16 @@ build_custom_linux() {
 
     checkout_repo "$LINUX_CUSTOM_DIR" "$LINUX_CUSTOM_URL" "$LINUX_CUSTOM_BRANCH"
 
-    cp $SRCDIR/resources/linux-config-aarch64 $LINUX_CUSTOM_DIR/.config
-
-    pushd $LINUX_CUSTOM_DIR
-    time make -j `nproc`
-    cp arch/arm64/boot/Image "$WORKLOADS_DIR/" || exit 1
-    cp arch/arm64/boot/Image.gz "$WORKLOADS_DIR/" || exit 1
-    popd
+    if [ ! -f "$LINUX_CUSTOM_DIR/.built.linux-custom" ]; then
+        cp $SRCDIR/resources/linux-config-aarch64 $LINUX_CUSTOM_DIR/.config
+
+        pushd $LINUX_CUSTOM_DIR
+        time make -j `nproc`
+        cp arch/arm64/boot/Image "$WORKLOADS_DIR/" || exit 1
+        cp arch/arm64/boot/Image.gz "$WORKLOADS_DIR/" || exit 1
+        touch .built.linux-custom
```

Comment:
You don't need to give different names to such indicators. Using an uniformed name `.built` could be simply and clean.

Reply:
Done